src/components/interaction-designer/block-editors/EndRecordingDigitsEditor.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div class="end-recording-digits-editor">
+    <validation-message
+      :message-key="`block/${block.uuid}/config/ivr/end_recording_digits`">
+      <template #input-control="{ isValid }">
+        <div v-if="hasIvr">
```

Comment:
Can we move the `v-if="hasIvr"` in the root element ? That way we won't render anything if `hasIvr` is false. 
I think we should have done that for `/block-editors/MaxDurationSecondsEditor.vue`, can I ask you to fix `MaxDurationSecondsEditor.vue` too ?

Context:
- before we didn't have that wrapper `<validation-message>`
- so, when we wrapped everything, we forgot to move such condition into the root element

Reply:
Pls, never mind about the MaxDurationSecondsEditor, I will fix it in my vmo-4326. Sorry

Reply:
(Pls ignore the **deleted** comment after this in case you received it in your email)